lib/livebook/teams/org.ex

```diff
@@ -0,0 +1,88 @@
+defmodule Livebook.Teams.Org do
+  @moduledoc false
+
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Livebook.Teams
+
+  @type t :: %__MODULE__{
+          id: pos_integer() | nil,
+          name: String.t() | nil,
+          teams_key: String.t() | nil,
+          user_code: String.t() | nil
+        }
+
+  embedded_schema do
+    field :name, :string
+    field :teams_key, :string
+    field :user_code, :string
+  end
+
+  @fields ~w(id name teams_key user_code)a
+
+  @doc """
+  Creates an Org.
+
+  With success, notifies interested processes about org creation.
+  Otherwise, it will return an error tuple with changeset.
+  """
+  @spec create_org(t(), map()) :: {:ok, map()} | {:error, Ecto.Changeset.t()}
+  def create_org(%__MODULE__{} = org, attrs) do
+    changeset = changeset(org, attrs)
+
+    case apply_action(changeset, :insert) do
+      {:ok, org} -> Teams.create_org(org)
+      {:error, changeset} -> {:error, Map.replace!(changeset, :action, :validate)}
+    end
+  end
```

Comment:
This is akin to a database interaction, so perhaps let's move it up to the `Teams` context?

Reply:
I followed the same pattern we have with the Hubs. But yeah, I can move ir up to `Teams` context